requirements-dev.txt

```diff
@@ -77,4 +77,6 @@ flake8-print==3.1.0
 pep8-naming==0.8.2
 
 # Check string quotes
-flake8-quotes==2.1.0
\ No newline at end of file
+flake8-quotes==2.1.0
+
+# Required by pytest and flake8-bugbear
```

Comment:
What is required by "pytest and flake8-bugbear"?

Reply:
Ignore this PR.